include/kafka/AdminClient.h

```diff
@@ -292,5 +323,51 @@ AdminClient::listTopics(std::chrono::milliseconds timeout)
     return Admin::ListTopicsResult(names);
 }
 
+inline Admin::DeleteRecordsResult
+AdminClient::deleteRecords(const TopicPartitionOffsets& tpos,
+                           std::chrono::milliseconds timeout) {
+    auto rk_queue = rd_kafka_queue_unique_ptr(rd_kafka_queue_new(getClientHandle()));
+
+    rd_kafka_DeleteRecords_unique_ptr rkDeleteRecords(rd_kafka_DeleteRecords_new(createRkTopicPartitionList(tpos)));
+    rd_kafka_DeleteRecords_t* rk_offsets = rkDeleteRecords.get();
+
+    rd_kafka_DeleteRecords(getClientHandle(), &rk_offsets, 1, nullptr, rk_queue.get());
+
+    auto rk_ev = rd_kafka_event_unique_ptr();
+
+    const auto end = std::chrono::steady_clock::now() + timeout;
+    do
+    {
+        rk_ev.reset(rd_kafka_queue_poll(rk_queue.get(), EVENT_POLLING_INTERVAL_MS));
+
+        if (rd_kafka_event_type(rk_ev.get()) == RD_KAFKA_EVENT_DELETERECORDS_RESULT) break;
+
+        if (rk_ev)
+        {
+            KAFKA_API_DO_LOG(Log::Level::Info, "rd_kafka_queue_poll got event[%s], with error[%s]", rd_kafka_event_name(rk_ev.get()), rd_kafka_event_error_string(rk_ev.get()));
```

Comment:
Here should be `Log::Level::Err`.
It's my bad, -- `AdminClient::createTopics`/`AdminClient::deleteTopics` is using the wrong log level (should be `Log::Level::Err` as well)

Reply:
Good catch. I changed the logging level here, additionally, I change the logging level for `AdminClient::createTopics`/ `AdminClient::deleteTopics` as well to `Log::Level::Err`. Please check it out and confirm if it is expected.